evm/src/main/java/org/hyperledger/besu/evm/internal/Words.java

```diff
@@ -94,6 +107,29 @@ static long clampedToLong(final Bytes uint) {
     }
   }
 
+  /**
+   * The value of the bytes as though it was representing an unsigned integer, however if the value
+   * exceeds Integer.MAX_VALUE then Integer.MAX_VALUE will be returned.
+   *
+   * @param uint the unsigned integer
+   * @return the least of the integer value or Integer.MAX_VALUE
+   */
+  static int clampedToInt(final Bytes uint) {
```

Comment:
I think this warrants some test coverage.

Reply:
Vectors added